src/components/color-picker/RuiColorInput.vue

```diff
@@ -0,0 +1,141 @@
+<script lang="ts" setup>
+import tinycolor from 'tinycolor2';
+import { clamp } from '@vueuse/shared';
+import RuiTextField from '@/components/forms/text-field/RuiTextField.vue';
+import RuiButton from '@/components/buttons/button/RuiButton.vue';
+import RuiIcon from '@/components/icons/RuiIcon.vue';
+import type { Color } from './utils';
+
+const props = defineProps<{
+  modelValue: Color;
+}>();
+
+const emit = defineEmits<{
+  (e: 'update:model-value', color: Color): void;
+}>();
+
+const modelValue = useVModel(props, 'modelValue', emit, {
+  eventName: 'update:model-value',
+});
```

Comment:
this is what triggered the other comment @lukicenturi do we still need the eventName here? or can we just remove it?

Reply:
yeah I will remove it